openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/TestCases.spec.ts

```diff
@@ -212,7 +212,7 @@ test('Custom SQL Query', async ({ page }) => {
     await test.step('Delete', async () => {
       await deleteTestCase(page, testCase.name);
     });
-  } catch (error) {
+  } finally {
```

Comment:
E2E test is already there but it was not failing as by mistake i have added catch block instead of finally